assets/scss/_variables_project.scss

```diff
@@ -1083,18 +1173,21 @@ div.td-page-meta__lastmod {
 
 // Update the sidebar navigation styles
 .td-sidebar-nav {
-    font-family: "Source Sans Pro", sans-serif;
+    font-family: $base-font;
     padding-left: 15px !important;
+    font-size: 14px !important; // Slightly larger than right nav
```

Comment:
This one is 14 as opposed to 13 above, and these are still navigation `<ul><li>`, right?

Reply:
Right-nav is now 12, so I think having the left-nav be 14 makes sense; page titles are shorter than headings generally and we frequently shorten them even more with the `linkTitle` front-matter for pure "scan-ability." This one I'm calling good.